src/dbus/server/dbus_agent.cpp

```diff
@@ -49,47 +49,58 @@ DBusAgent::DBusAgent(otbr::Ncp::ControllerOpenThread &aNcp)
 
 void DBusAgent::Init(void)
 {
-    DBusError   dbusError;
-    otbrError   error = OTBR_ERROR_NONE;
-    int         requestReply;
-    std::string serverName = OTBR_DBUS_SERVER_PREFIX + mInterfaceName;
+    otbrError error = OTBR_ERROR_NONE;
 
-    dbus_error_init(&dbusError);
-    DBusConnection *conn                = nullptr;
-    auto            connection_deadline = std::chrono::steady_clock::now() + std::chrono::seconds(30);
+    auto connection_deadline = std::chrono::steady_clock::now() + std::chrono::seconds(30);
```

Comment:
```suggestion
    auto connection_deadline = Clock::now() + Seconds(30);
```

consider using the `Clock` type alias in `common/time.hpp` to enforce always using `steady_clock`.
```c++
using Seconds      = std::chrono::seconds;
using Milliseconds = std::chrono::milliseconds;
using Microseconds = std::chrono::microseconds;
using Clock        = std::chrono::steady_clock;
using Timepoint    = Clock::time_point;
```

Reply:
Added `using Clock        = std::chrono::steady_clock;`

But other suggestions are not accepted.